docs/content/Getting Started Using.md

```diff
@@ -126,4 +142,48 @@ dotnet msbuild /t:AnalyzeSolution
 
 Note: we passed the `--code-root` flag so that the `*.sarif` report files will report file paths relative to this root. This can be imported for certain editors to function properly. 
 
+## MSBuild tips and tricks
+
+MSBuild can be overwhelming for the uninitiated. Here are some tricks we've seen in the wild:
+
+### Use well-known properties
+
+Checkout the [MSBuild reserved and well-known properties](https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-reserved-and-well-known-properties?view=vs-2022) to use existing variables like `$(MSBuildProjectFile)`.
+
+### Wrap path arguments in quotes
+
+As MSBuild is all XML, you can use `&quot;` to wrap evaluated values in quotes:
+
+```xml
+<PropertyGroup>
+    <WithQuotes>&quot;$(SolutionDir)&quot;</WithQuotes>
+</PropertyGroup>
+```
+
+### Extend `<FSharpAnalyzersOtherFlags>` in multiple lines
+
+You can extend the value of `$(FSharpAnalyzersOtherFlags)` by setting it again in multiple lines:
+
+```xml
+<PropertyGroup>
+    <FSharpAnalyzersOtherFlags>--analyzers-path &quot;$(PkgG-Research_FSharp_Analyzers)/analyzers/dotnet/fs&quot;</FSharpAnalyzersOtherFlags>
+    <FSharpAnalyzersOtherFlags>--analyzers-path &quot;$(PkgIonide_Analyzers)/analyzers/dotnet/fs&quot;</FSharpAnalyzersOtherFlags>
```

Comment:
Don't we neet the `$(FSharpAnalyzersOtherFlags)` prefix here, too?

Reply:
Yup, good catch! Thanks!